easy_phi/app.py

```diff
@@ -80,6 +91,7 @@ def prepare(self):
         if err:
             self.set_status(400)
             self.finish({'error': err})
+            return
```

Comment:
it will cause merge conflict and I will have to update manually. In other words, it is not a problem